core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3241,9 +3241,9 @@ public void activeSyncMetadata(AlluxioURI path, Collection<AlluxioURI> changedFi
         executorService.invokeAll(callables);
       }
     } catch (InvalidPathException e) {
-      LOG.warn("InvalidPathException during active sync {}", e);
+      LOG.warn("InvalidPathException during active sync {}", e.getMessage());
```

Comment:
`e.toString` is better than `e.getMessage`

Reply:
Oh, I was following guide here: https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html?q=getMessage#warn-log-level. Do we want to update the guide too?

Reply:
Yeah we should. The issue with `.getMessage` is sometimes the message is not populated well, and `.toString` is generally just the exception type + message.